### PR TITLE
content type fix

### LINKED ISSUE #192: "Response had no 'Content-Type' header" warning when sending CSS
When I have a simple Vapor server with an index file and CSS in `Public/stylesheets`, every time the page is loaded the server logs a warning because no `Content-Type` was attached.

```
[1461329360] [INFO] GET /
[1461329360] [INFO] GET /stylesheets/style.css
[1461329360] [WARNING] Response had no 'Content-Type' header.
```

We can do either
- detect a couple of basic files usually present in Public and attach the correct content type (recommended)
- ignore the warning and disable it

Thoughts?

### PR DESCRIPTION
**Pull request by tannernelson**
Deduces content types for a given file extension.

For example:

```
"css": "text/css"
```

Fixes #192

### CODE REVIEW

File: Sources/Vapor/Core/Application.swift [hunk 1]
```diff
@@ -237,7 +238,16 @@ public class Application {
         // File exists
         if let fileBody = try? FileManager.readBytesFromFile(filePath) {
             return Request.Handler { _ in
-                return Response(status: .ok, headers: [:], body: Data(fileBody))
+                var headers: Response.Headers = [:]
+
+                if
+                    let fileExtension = filePath.split(byString: ".").last,
```
Comment (LoganWright) [inline Sources/Vapor/Core/Application.swift]: I always want this to be `func split(by string:`? Do we own it

Comment (tannernelson) [inline Sources/Vapor/Core/Application.swift]: It's from Zewo/String. We can make a PR there.

File: Tests/Vapor/ApplicationTests.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+@testable import Vapor
+
+class ApplicationTests: XCTestCase {
+    static var allTests: [(String, ApplicationTests -> () throws -> Void)] {
+        return [
+            ("testMediaType", testMediaType),
+        ]
+    }
+
+    var workDir: String {
```
Comment (LoganWright) [inline Tests/Vapor/ApplicationTests.swift]: YES! Finally!

File: Tests/Vapor/ApplicationTests.swift [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        guard let response = try? app.respond(to: request) else {
+            XCTFail("App could not respond")
+            return
+        }
+
+        var found = false
+        for header in response.headers {
+            if header.key == "Content-Type" && header.value == "text/css" {
+                found = true
+            }
```
Comment (LoganWright) [inline Tests/Vapor/ApplicationTests.swift]: break?

File: Tests/Vapor/ApplicationTests.swift (hunk 2 above)
Comment (tannernelson) [inline Tests/Vapor/ApplicationTests.swift]: I thought you added that! It just appeared in the Config test haha.

File: Sources/Vapor/Core/Application.swift [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import libc
+import MediaType
 
 public class Application {
     public static let VERSION = "0.5.3"
```
Comment (czechboy0) [inline Sources/Vapor/Core/Application.swift]: Can you please update the version?

File: Tests/Vapor/ApplicationTests.swift (hunk 2 above)
Comment (czechboy0) [inline Tests/Vapor/ApplicationTests.swift]: 👋

File: Sources/Vapor/Core/Application.swift (hunk 4 above)
Comment (tannernelson) [inline Sources/Vapor/Core/Application.swift]: Oops, yeah. Looks like we've been forgetting to do that. I'll make an issue for the 0.8 release (milestone).

**Pull request merged**